buildkite/src/Jobs/Test/RosettaIntegrationTests.dhall

```diff
@@ -33,19 +35,20 @@ Pipeline.build
       JobSpec::{
         dirtyWhen = dirtyWhen,
         path = "Test",
-        name = "RosettaIntegrationTests"
+        name = "RosettaIntegrationTests",
+        tags = [ PipelineTag.Type.Long, PipelineTag.Type.Test ]
```

Comment:
Does this mean this test is not run on `!ci-build-me`? This fast test version should run on every PR when dirty as it can catch some breaking changes like archive schema changes.

Reply:
It will. This is further improvements for pipeline itself. It tells you about order of tests. In theory first we ran `Fast` tests then `Long` . When any of fast test will fail then we don't run Long test and mark pipeline as failed. If all of Fast tests are green then we ran Long tests. Thanks to that we skip running heavy tests on PR which we now is breaking CI. 

This is small revolution so currently pipeline is configured with backward compatibility (it will ran All of tests Fast/Long etc without any internal dependencies)

In future i would like to have such triaging implemented in pipeline to preserve gcloud resources and provide faster feedback.

Reply:
Thank you for your explanation.